applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_ingresscontroller/rule.yml

```diff
@@ -37,5 +37,5 @@ template:
     filepath: '/apis/operator.openshift.io/v1/namespaces/openshift-ingress-operator/ingresscontrollers/default'
     yamlpath: ".status.tlsProfile.ciphers[:]"
     values:
-    - value: '^(ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-RSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305|ECDHE-RSA-AES256-GCM-SHA384|ECDHE-RSA-CHACHA20-POLY1305|ECDHE-ECDSA-AES256-GCM-SHA384|AES256-GCM-SHA384|AES128-GCM-SHA256)$'
+    - value: '^(ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-RSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305|ECDHE-RSA-AES256-GCM-SHA384|ECDHE-RSA-CHACHA20-POLY1305|ECDHE-ECDSA-AES256-GCM-SHA384|TLS_AES_128_GCM_SHA256|TLS_AES_256_GCM_SHA384|TLS_CHACHA20_POLY1305_SHA256|DHE-RSA-AES128-GCM-SHA256|DHE-RSA-AES256-GCM-SHA384)$'
```

Comment:
As noted by @sluetze - we need to update the ocil above to be consistent with this.

Reply:
also the remediation needs to be updated, I just recognized (while doing other work with this rule at the moment) that I get
```
status:
  applicationState: Error
  errorMessage: 'Unable to get fix object from ComplianceRemediation. Please update
    the compliance-operator''s permissions: ingresscontrollers.operator.openshift.io
    "default" is forbidden: User "system:serviceaccount:openshift-compliance:compliance-operator"
    cannot get resource "ingresscontrollers" in API group "operator.openshift.io"
    in the namespace "openshift-ingress-operator"'
```

Reply:
I have a PR to fix the permissions in the operator:

https://github.com/ComplianceAsCode/compliance-operator/pull/558